### DISCUSSION TITLE
How do i obfuscate library, build it and publish it on npm?

### DISCUSSION THREAD

**Discussion by amansharmawmp** [callstack/react-native-builder-bob#416] [General]
I've created a library and i need to obfuscate its src folder. How can i achieve that? I tried javascript-obfuscator library, it did obfuscated the code but it won't get run in react-native app, error "dynamic import not supported by metro" was thrown

**Comment by cwoolum** [comment]
Javascript obfuscation isn't actually meaninigful... You get the same level of indirection by just minifying. If you really want to hid your code, you'll need to use something like wasm.
